variables.tf

```diff
@@ -109,7 +115,7 @@ variable "security_group" {
       object({
         name      = string
         direction = string
-        source    = string
+        remote    = string
```

Comment:
Could we just make sure to mark that it is a breaking change in the PR - we'll need to bump the major version here.